manifest: sdk-nrfxlib: nrf_fuel_gauge: Update to v2.0.0 (#28924)
nrfconnect/sdk-nrf: samples/pmic/native/npm2100_fuel_gauge/README.rst

| :kconfig:option:`CONFIG_BATTERY_MODEL_LITHIUM_CR2032` | Lithium CR2032 coin cell                  |
+-------------------------------------------------------+-------------------------------------------+

> When the Zephyr Settings subsystem is enabled, the sample also demonstrates how to store the fuel gauge state in non-volatile memory and restore it on boot. Fuel gauge state can be stored using the shell command described in `Testing`_.

Reviewer: ```suggestion
When the Zephyr Settings subsystem is enabled, the sample also demonstrates how to store the fuel gauge state in non-volatile memory and restore it on boot.
You can store the fuel gauge state using the shell command described in `Testing`_.
```

   $ battery_model Lithium_CR2032
   Fuel gauge initialised for Lithium CR2032 battery.

To store the fuel gauge state in non-volatile memory, use the following shell command:

.. code-block:: console

   $ fuel_gauge_state_store
   Storing state after next update

> Fuel gauge state will be stored in non-volatile memory after the next measurement has been made, and restored on the next boot.

Reviewer: ```suggestion
The fuel gauge state will be stored in non-volatile memory after the next measurement has been performed, and restored on the next boot.
```

> The state is stored per battery type, so changing the battery type with the ``battery_model`` command will not affect the stored state of other battery types.

Reviewer: ```suggestion
The state is stored for each battery type, so changing the battery type with the ``battery_model`` command will not affect the stored state of other battery types.
```

> Use the following shell commands to view the stored state and clear it, using Alkaline_AA as an example:

Reviewer: ```suggestion
Use the following shell commands to view the stored state and clear it, using Alkaline AA as an example:
```
